packages/react/src/SuspensiveProvider.ko.mdx

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 6
+title: SuspensiveProvider
+---
+
+이 컴포넌트는 Suspensive가 제공하는 컴포넌트들(Suspense, Delay 등)의 설정을 한 번에 제어합니다.
+
+```tsx
+import { Suspensive, SuspensiveProvider } from '@suspensive/react'
+
+const suspensive = new Suspensive({
+  defaultOptions: {
+    delay: {
+      ms: 1200,
+    },
+    suspense: {
+      fallback: <Spinner />,
+    },
+  },
+})
+
+function App() {
+  return <SuspensiveProvider value={suspensive}>...</SuspensiveProvider>
```

Comment:
I have a question.!

If we are using both the `Delay` component and `Suspense` at the same time within the SuspensiveProvider,
like below code
```tsx

<SuspensiveProvider value={ms:2000}> // pseudo code
  <Suspense>
    const {data} =  useSuspenseQuery()
  </Suspense>
</SuspensiveProvider>

```

It takes 2000ms? or 2000ms + api call(network)


Does Suspense's asynchronous call only occur when Delay ends?

Reply:
```tsx
const suspensive = new Suspensive({
  defaultOptions: {
    delay: {
      ms: 2000
    },
    suspense: {
      fallback: <Spinner /> 
    }
  }
})

<SuspensiveProvider value={suspensive}> // pseudo code
  <Delay>  // not showing during 2000ms and then below component will be showen
    <Suspense>  // after delayed by parent Delay, api will be called. then during api pending, default fallback Spinner(by Suspensive's defaultOptions) will show
      const {data} =  useSuspenseQuery()
    </Suspense>
  </Delay>
</SuspensiveProvider>
```

but I want to make users use like below.

```tsx
const suspensive = new Suspensive({
  defaultOptions: {
    delay: {
      ms: 2000
    }
  }
})

<SuspensiveProvider value={suspensive}> // pseudo code
  <Suspense fallback={<Delay><Spinner /></Delay>}>
    const {data} =  useSuspenseQuery()
  </Suspense>
</SuspensiveProvider>
```

Reply:
After I coding, I felt defaultOptions is not options. just `defaultProps`?

Reply:
Thank you for the detailed explanation.!

I think option naming is good as it is(`defaultOptions`)
They don't go into the components, they go into the class called new Suspensive